.gitlab-ci.yml

```diff
@@ -222,33 +222,28 @@ publish-linux-snap-amd64:          &publish_snap
     BUILD_ARCH:                    amd64
   dependencies:
     - package-linux-snap-amd64
+  allow_failure:                   true
```

Comment:
How is this possible, we are building nightly only once per day. This should ideally not fail. Can't we fix this on snap side?

Reply:
The upload only fails for uploads of exactly the same snap for the same version/architecture. So if there are no changes in the code nightly builds will still produce a snap which will be upload and hash check on snapcraft refuses it then. I found no way of changing this behaviour.

IMHO the easiest way to fix this would be to have a version like the current timestamp (e.g. 20180905) or git commit like before: https://imgur.com/a/bgpoG7T

Reply:
The version is wrong anyways, this is not 2.1.0. This need to be fixed in the script that figures out the version. Adding a commit hash and date would help here if `nightly`.